example/urls.py

```diff
@@ -1,7 +1,6 @@
-from django.conf.urls import patterns
+from django.urls import path
```

Comment:
I use this project in Django 1.11, which is supported until 2020. So I would prefer that this used `from django.conf.urls import url` and below changed to use `url`, thanks :)

Reply:
Note this change is made in https://github.com/dracos/django-date-extensions/pull/17 (which also sets up testing of 1.11) which needs merging.